build/salesforce/requirements/airbyte-interop/src_dbt_requirements.yml

```
@@ -0,0 +1,1130 @@
+# This file was auto-generated using the morph cli.
+# Please do not edit manually.
+version: 2
+
+sources:
+  - name: salesforce
+    schema: "{{ var('salesforce_schema', 'salesforce') }}" # you will need to customize if your schema name is not salesforce
+    database: "{% if target.type != 'spark'%}{{ var('salesforce_database', target.database) }}{% endif %}"
+    loader: fivetran
+    loaded_at_field: _fivetran_synced 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)